cluster/environments/azure-multiple-clusters/aks-centralus.tf

```diff
@@ -7,83 +7,74 @@ resource "azurerm_resource_group" "centralrg" {
+  gitops_poll_interval     = "${var.gitops_poll_interval}"
+  resource_group_location  = "${var.central_resource_group_location}"
+  resource_group_name      = "${azurerm_resource_group.centralrg.name}"
   service_cidr             = "${var.central_service_cidr}"
-  dns_ip                   = "${var.central_dns_ip}"
-  docker_cidr              = "${var.central_docker_cidr}"
-  ssh_public_key           = "${var.ssh_public_key}"
   service_principal_id     = "${var.service_principal_id}"
   service_principal_secret = "${var.service_principal_secret}"
-  kubeconfig_recreate      = ""
+  ssh_public_key           = "${var.ssh_public_key}"
+  vnet_subnet_id           = "${module.central_vnet.vnet_subnet_ids[0]}"
+  dns_ip                   = "${var.central_dns_ip}"
+  docker_cidr              = "${var.central_docker_cidr}"
   kubeconfig_filename      = "${local.central_kubeconfig_filename}"
 }
 
-# Deploys flux in aks cluster
-module "central_flux" {
-  source = "../../common/flux"
-
-  gitops_ssh_url      = "${var.gitops_ssh_url}"
-  gitops_ssh_key      = "${var.gitops_ssh_key}"
-  flux_recreate       = ""
-  kubeconfig_complete = "${module.central_aks.kubeconfig_done}"
-  kubeconfig_filename = "${local.central_kubeconfig_filename}"
-  flux_clone_dir      = "${local.central_flux_clone_dir}"
-  gitops_path         = "${var.gitops_central_path}"
-  gitops_poll_interval = "${var.gitops_poll_interval}"
-}
-
 # create a static public ip and associate with traffic manger endpoint
 module "central_tm_endpoint" {
   source = "../../azure/tm-endpoint-ip"
 
-  resource_group_name                 = "${var.service_principal_is_owner == "1" ? local.central_rg_name : module.central_aks.cluster_derived_resource_group}"
+  resource_group_name                 = "${local.central_rg_name}"
   resource_location                   = "${local.central_rg_location}"
   traffic_manager_resource_group_name = "${var.traffic_manager_resource_group_name}"
   traffic_manager_profile_name        = "${var.traffic_manager_profile_name}"
-  endpoint_name                       = "${local.central_rg_location}-${var.cluster_name}"
+  endpoint_name                       = "${local.central_rg_location}_${var.cluster_name}"
   public_ip_name                      = "${var.cluster_name}"
   ip_address_out_filename             = "${local.central_ip_address_out_filename}"
 
   tags = {
-    environment = "azure-multiple-clusters - ${var.cluster_name} - public ip"
-    kubedone = "${module.central_aks.kubeconfig_done}"
+    environment = "azure_multiple_clusters - ${var.cluster_name} - public ip"
+    kubedone = "${module.central_aks_gitops.kubeconfig_done}"
   }
 }
 
 # Create a role assignment with Contributor role for AKS client service principal object
 #   to join vnet/subnet/ip for load balancer/ingress controller
 resource "azurerm_role_assignment" "central_spra" {
-  count                = "${var.service_principal_is_owner == "1" ? 1 : 0}"
```

Comment:
Since we are removing `service_principal_is_owner`, the documentation needs to be updated as well.  It appears the only doc is `environments/azure-multiple-clusters/README.md`